libraries/AP_HAL_ChibiOS/RCOutput.h

```diff
@@ -644,12 +666,15 @@ class ChibiOS::RCOutput : public AP_HAL::RCOutput
     uint16_t create_dshot_packet(const uint16_t value, bool telem_request, bool bidir_telem);
     void fill_DMA_buffer_dshot(dmar_uint_t *buffer, uint8_t stride, uint16_t packet, uint16_t clockmul);
 
+    // event to allow dshot cascading
```

Comment:
ifdef so DSHOT_CASCADE not exist or is zero on non-IOMCU dshot

Reply:
Fixed